gateway/1.4/modules/ROOT/pages/flex-view-replica-status.adoc

```diff
@@ -5,21 +5,49 @@ endif::[]
 :page-aliases: runtime-manager::flex-gateway-replica-status.adoc
 :imagesdir: ../assets/images
 
-To view information about the status of Flex Replicas:
+In Runtime Manager, you can view the status of groups of Flex Replicas and individual Flex Replicas.
+
+== Groups of Flex Replicas
+
+To view information about the status of groups of Flex Replicas:
 
 . From Anypoint Platform, select *Runtime Manager*.
-. Select *Flex Gateway* in the navigation menu.
+. Select *Flex Gateway* in the navigation menu. +
+By default, all Flex Gateways regardless of their status are shown. To filter by status, click the *Any Status* arrow and the status you want to filter.
+
+The *Name* column lists the name and the number of connected replicas.
+
+The *Status* column shows the connection status for the replicas associated with a particular gateway.
+
+The following table describes the meaning of each status for a group of replicas:
+
+[%header,cols="30a,70a"]
+|===
+|Status | Group of Replicas
+|Connected | At least one replicas is to the platform and are serving traffic.
+|Disconnected | There are no replicas connected to the platform and no information on whether they are serving traffic.
+|===
+
+== Individual Flex Replicas
 
-The *Status* column in the list of gateways shows the connection status for the replicas associated with a particular gateway.
+To view information about the status of individual replicas:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the navigation menu.
+. Select the name of the Flex Gateway whose individual replica statuses you want to view. +
+By default, disconnected replicas are not shown. To view disconnected replicas, select *Show disconnected Replicas*. 
 
 The *Status* column in the list of replicas shows the connection status for that particular replica.
 
-The following table describes the meaning of each status for a single replica or group of replicas:
+The following table describes the meaning of each status for a single replica:
 
-[%header,cols="20a,30a,30a"]
+[%header,cols="30a,70a"]
+|===
+|Status |Single Replica
+|Connected | The replica is connected to the platform and is serving traffic.
+|Disconnected |The replica is not connected to the platform. There is no information on whether it is serving traffic. Disconnected replicas are deleted after seven days.
```

Comment:
Same comment as above for Gateways (If the replica is not connected to the platform, they will not be serving traffic. So, not sure why we say "and no information on whether they are serving traffic.")

Reply:
There are other columns on the Flex Replicas Page, again not sure why they are not being mentioned here.